examples/advanced-web-service/__snapshots__/chart.test.ts.snap

```diff
@@ -735,6 +737,8 @@ server {
         allow 172.16.0.0/12;
         deny all;
     }
+
+    
```

Comment:
comment: This now leaves a bit of whitespace in the config. This shouldn't cause a problem for nginx but it might be "weird" to passers by?